cura/API/Account.py

```diff
@@ -90,11 +90,23 @@ def _onLoginStateChanged(self, logged_in: bool = False, error_message: Optional[
             self.loginStateChanged.emit(logged_in)
 
     @pyqtSlot()
-    def login(self) -> None:
+    @pyqtSlot(bool)
+    def login(self, force_logout_before_login: bool = False) -> None:
+        """
+        Initializes the login process. If the user is logged in already and force_logout_before_login is true, Cura will
+        logout from the account before initiating the authorization flow. If the user is logged in and
+        force_logout_before_login is false, the function will return, as there is nothing to do.
+
+        :param force_logout_before_login: Optional boolean parameter
```

Comment:
This doesn't add any information beyond what is visible from the signature. I would leave it out next time